### ISSUE TITLE
In SSD Model, why do we add a softmax layer in test stage?

### ISSUE THREAD

**Issue by DayChan** [qfgaohao/pytorch-ssd#40]
Why do we add a Softmax Layer in test stage? It seems strange because the softmax layer is not trained.

**Comment by shtern** [comment]
@DayChan excuse me, did you find the answer to this question? This layer also causes problems when converting the model to CoreML
